.github/workflows/release.yml

```diff
@@ -135,7 +135,22 @@ jobs:
       - name: Push images with version tag
         env:
           CTR_TAG: ${{ needs.version.outputs.version }}
-        run: make docker-push
+          IMAGE_REPO: openservicemesh
+          IMAGE_TAG_LIST: '["bookbuyer","bookthief","bookstore","bookwarehouse","tcp-echo-server","tcp-client","init","osm-controller"]'
+        run: |
+          for image_tag in $(echo "${IMAGE_TAG_LIST}" | jq -r '.[]')
```

Comment:
Making this a JSON blob seems unnecessary. I would probably do something like
```bash
for image_tag in bookbuyer bookthief bookstore ...
```
and skip the environment variable if it's only referred to once.